src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -92,7 +92,7 @@ public function getItemFromIri(string $iri, array $context = [])
     public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH): string
     {
         $resourceClass = $this->getObjectClass($item);
-        $routeName = $this->routeNameResolver->getRouteName($resourceClass, false);
+        $routeName = $this->routeNameResolver->getRouteName($resourceClass, OperationType::ITEM, []);
```

Comment:
Haha no we can't do this because it'd need to change the interface, which becomes then a breaking change (ie major version). Leave the hack alone, it'll be removed in 3.x (hopefully).